recipe/meta.yaml

```diff
@@ -51,8 +51,8 @@ build:
     - /usr/lib/libresolv.9.dylib  # [osx]
     - /usr/lib/libsandbox.1.dylib  # [osx]
     - '/System/Library/Frameworks/CoreLocation.framework/**'  # [osx]
-    #- '*/api-ms-win-core-winrt-l1-1-0.dll'  # [win]
-    #- '*/api-ms-win-core-winrt-string-l1-1-0.dll'  # [win]
+    - '*/api-ms-win-core-winrt-l1-1-0.dll'  # [win]
```

Comment:
are those related to icu change ?

Reply:
it was to fix a missing dso warning, but JC already explained why it was needed, I reverted this